packages/components/src/ui/color-picker/styles.ts

```diff
@@ -91,8 +91,15 @@ export const ColorfulWrapper = styled.div`
 	.react-colorful__pointer {
 		height: 16px;
 		width: 16px;
-		border: ${ CONFIG.borderWidthFocus } solid rgba( 255, 255, 255, 0 );
-		box-shadow: inset 0px 0px 0px ${ CONFIG.borderWidthFocus } #ffffff;
+		border: none;
+		box-shadow: 0 0 2px 0 rgba( 0, 0, 0, 0.25 );
```

Comment:
Would it make sense to use ${ CONFIG.radiusBlockUi } (2px) instead of hardcoded 2px?

Reply:
Yes! I'll make that happen.

Reply:
Well, sorry no, that makes sense for replacing `border-radius: 2px`, but not for the shadow. We want a separate box shadow value for this one, something a la ${ CONFIG.elevation1 } or something.

Reply:
Got it, in that case feel free to merge the PR 👍

Reply:
Thanks. Can I get a green check?